### DISCUSSION TITLE
separateMultiplePatch implementation?

### DISCUSSION THREAD

**Discussion by faenschi** [renovatebot/renovate#30589] [Suggest an Idea]
### Tell us more.

Hi everyone,

Are there any plans to implement the `separateMultiplePatch` feature as discussed in this [GitHub issue](https://github.com/renovatebot/renovate/issues/6576)?

Best regards,  
Angela

**Comment by rarkins** [comment]
Not planned. What's your use case for this? It would mean a separate PR for every single version, which wouldn't be a good idea in most cases.
